main_test.go

```diff
@@ -264,3 +264,27 @@ func backUpDefaultDistro() (func(), error) {
 	}
 	return restore, nil
 }
+
+// keepAwake sends an endless command to the distro to keep it awake.
+// The distro will stay awake until the context is cancelled or the cancel
+// function is called.
+//
+// You must call the cancel function to release the associated resources.
+func keepAwake(ctx context.Context, d *wsl.Distro) (context.CancelFunc, error) {
```

Comment:
Very elegant. My only question is why you don’t take a `*testing.T` here and make it a test helper. That way, you don’t return an error and fail the test early… Which then reduce the caller size (no check for error needed)

Reply:
Agreed, changing it.

Reply:
Hmm this could start a war. Choose your team:
```
keepAwake(ctx context.Context, t *testing.T, d *wsl.Distro)
keepAwake(t *testing.T, ctx context.Context, d *wsl.Distro)
```

Reply:
The linters prefer the first one

Reply:
pfffff, those linters… let it wins then (even if spontenously, I would have gone with the second form)…

Reply:
Well I knew the linter complains because I went for the second form first as well haha. Anyways, let's submit to linter.

Reply:
reading it, and especially the calls, this blows my mind. I think for test helpers, testing.T should really be first, do you mind flip flopping + relax the linter for this helper (with a small rationale like "this is a test helper")?